test.js

```diff
@@ -32,6 +32,17 @@ test('nanomorph', (t) => {
       t.equal(String(res), expected, 'result was expected')
     })
 
+    t.test('should morph a node with namespaced attribute', (t) => {
+      t.plan(1)
+
+      const oldTree = html`<svg><use xlink:href="#heybooboo"></use></svg>`
```

Comment:
usage of template strings in tests is :+1: tho - we can assume folks run newer node versions when checking it out locally. Also this is `choo` code and `yo-yoify` should take care of it (: